### PR TITLE
[Admin] Sylius new version notification widget

### PR DESCRIPTION
**Pull request by NeverResponse**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | yes |
| BC breaks?      | no |
| Related tickets | - |
| License         | MIT |

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
Looks really nice, I am only not sure about the twig extension for the purpose of checking if notifications are enabled. I have 2 quick alternative solutions coming to my mind:

1. Twig globals;
2. Checking in the controller and rendering empty response.

Not sure if much better, but maybe will give you some extra ideas.

File: src/Sylius/Bundle/AdminBundle/Resources/config/services/controller.xml [hunk 1]
```diff
@@ -12,6 +12,10 @@
 -->
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <parameter key="sylius.admin.notification.uri">http://www.sylius-hub.org/sylius/version</parameter>
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/AdminBundle/Resources/config/services/controller.xml]: hub.sylius.com/version

**Comment by pjedrzejewski** [comment]
@NeverResponse Please update the URL and it is good to go. :)

File: src/Sylius/Bundle/AdminBundle/Resources/config/services/controller.xml (hunk 1 above)
Comment (pamil) [inline src/Sylius/Bundle/AdminBundle/Resources/config/services/controller.xml]: Are we sure about using sylius.**com**? Currently it redirects to sylius.**org** :)

Comment (NeverResponse) [inline src/Sylius/Bundle/AdminBundle/Resources/config/services/controller.xml]: @pjedrzejewski bump

Comment (pjedrzejewski) [inline src/Sylius/Bundle/AdminBundle/Resources/config/services/controller.xml]: Yes! :)

**Comment by pjedrzejewski** [comment]
Thanks Jasiek!

**Pull request merged**